CHANGELOG.asciidoc

```diff
@@ -88,6 +121,151 @@ https://github.com/elastic/beats/compare/v8.7.1...v8.8.0[View commits]
 
 - Add `event.category` and `event.type` to Sysmon module for EventIDs 8, 9, 19, 20, 27, 28, 255. {pull}35193[35193]
 
+[[release-notes-8.7.1]]
+=== Beats version 8.7.1
+https://github.com/elastic/beats/compare/v8.7.0\...v8.7.1[View commits]
+
+==== Bugfixes
+
+*Affecting all Beats*
+
+- Fix Beats started by agent not respecting the allow_older_versions: true configuration flag {issue}34227[34227] {pull}34964[34964]
+- Beats started by agent now configure one global instance of each default add_x_metadata processor instead of creating one processor per input. Fixes a significant performance degradation for the aws-s3 input. {issue}35000[35000] {pull}35031[35031]
```

Comment:
```suggestion
- Fix Beats started by Elastic Agent not respecting the `allow_older_versions: true` configuration flag {issue}34227[34227] {pull}34964[34964]
- Beats started by Elastic Agent now configures one global instance of each default `add_x_metadata` processor instead of creating one processor per input. Fixes a significant performance degradation for the aws-s3 input. {issue}35000[35000] {pull}35031[35031]
```